src/test/java/seedu/address/model/attribute/EmailTest.java

```diff
@@ -48,16 +48,22 @@ public void isValidEmail() {
         assertFalse(Email.isValidEmail("peterjack@example.com.")); // domain name ends with a period
         assertFalse(Email.isValidEmail("peterjack@-example.com")); // domain name starts with a hyphen
         assertFalse(Email.isValidEmail("peterjack@example.com-")); // domain name ends with a hyphen
+        assertFalse(Email.isValidEmail("peterjack-@example.com")); // local part ends with a hyphen
+        assertFalse(Email.isValidEmail("peterjack-@example..com")); // two periods consecutively
```

Comment:
The second test does not guarantee testing for two consecutive periods, since the local part ends with a hyphen. I would suggest removing the hyphen from the test for two periods.

Reply:
Done